backend/test/getTransactionHistory.test.ts

```diff
@@ -0,0 +1,207 @@
+import { PostgreSqlContainer, type StartedPostgreSqlContainer } from '@testcontainers/postgresql';
+import type { FastifyInstance } from 'fastify';
+import { afterAll, beforeAll, beforeEach, describe, expect, test } from 'vitest';
+
+import { buildApp } from '@/app';
+
+import * as schema from '../src/db/schema';
+
+function generateMockData(
+  fromAddress: string,
+  upperLimit: number,
+  lowerLimit?: number,
+  isResponseMock?: boolean,
+) {
+  const mockData = [];
+  const numEntries = upperLimit;
+  const startCnt = lowerLimit ? lowerLimit : 1;
+
+  for (let i = startCnt; i <= numEntries; i++) {
+    const transferId = `transfer_id_${i}`;
```

Comment:
transfer id is defined here please use the same format
 https://github.com/keep-starknet-strange/vault/blob/3150d944f48cf61f720985606a8d85b2b9017c67/infra/indexers/src/usdc-transfers.indexer.ts#L34